### PR TITLE
Add testing files

### PR DESCRIPTION
**Pull request by ldr426**
This PR aims to add testing files.

### CODE REVIEW

**Review by valentina-s — APPROVED** [review]
I have a few small inline comments. Once it is merged we can see if the testing workflow goes through. You can also set up status badges.

File: echoshader/tests/test_track.py
Comment (ldr426) [reply echoshader/tests/test_track.py:39]: > The order for tricolor is BGR: i.e. red 120, green 38, blue 18.

What does this mean? Could you explain a bit more

Comment (valentina-s) [reply echoshader/tests/test_track.py:39]: @ldr426 The tricolor colormap maps 120kHz channel to red channel, 38 kHz channel to green channel, and 18Hz channel to the blue one. So, for RGB the order will be 
```
MVBS_ds.eshader.echogram(
            channel=[
                "GPT 120 kHz 00907205a6d0 4-1 ES120-7C",
                "GPT  38 kHz 009072058146 2-1 ES38B",
                "GPT  18 kHz 009072058c8d 1-1 ES18-11",
            ],
```
Of course as a function, one could pass any order, but this would produce meaningful plot. Use this for the demo notebooks too.

File: echoshader/tests/test_hist.py
Comment (ldr426) [reply echoshader/tests/test_hist.py:36]: I think we should reserve both the multi-frequency echogram and tricolor in integration panel testing, given that they have unified behavior(box select)

Comment (valentina-s) [reply echoshader/tests/test_hist.py]: @ldr426 I agree they should be both tested. You can test separately the integration of the multi-frequency and the tricolor. For example, `test_track_integration` (this one tests with multi-frequency) and `test_track_tricolor_integration` (for tricolor  track integration).

Comment (ldr426) [reply echoshader/tests/test_hist.py]: Ok @valentina-s  Please check it

**Review by valentina-s — APPROVED** [review]
Thanks, looks good! Will merge and see if CI works fine.

**Pull request merged**
